src/pages/AddItem.js

```diff
@@ -1,29 +1,32 @@
-import React, { useState, useContext } from 'react';
+import React, { useState, useContext, useEffect } from 'react';
 import { withFirestore } from 'react-firestore';
 import AddItemHeader from './AddItemHeader';
 import './AddItem.css';
-import normalizeName from '../lib/normalizeName';
 import ItemError from './ItemError';
 import { ListContext } from '../listContext';
 
 const expectedPurchase = { soon: 7, kindOfSoon: 14, notSoon: 30 };
 
 const AddItem = ({ firestore }) => {
-  const { token, setShoppingList, shoppingList, fetchList } = useContext(
-    ListContext,
-  );
+  const {
+    token,
+    duplicate,
+    checkForDuplicates,
+    shoppingList,
+    fetchList,
+  } = useContext(ListContext);
 
   const [name, setName] = useState('');
   const [nextExpectedPurchase, setNextExpectedPurchase] = useState(0);
-  const [duplicate, setDuplicate] = useState(false);
 
+  useEffect(() => {
```

Comment:
This will be taken out. I left it in so I can chat with @mikeramz86 about what I was thinking to debug this.